src/compound-config/compound-config.cpp

```diff
@@ -533,13 +537,39 @@ CompoundConfig::CompoundConfig(const char* inputFile) {
     YConfig = YAML::Load(f);
     root = CompoundConfigNode(nullptr, YConfig, this);
     useLConfig = false;
+    f.close();
+    std::cout << "file closed" << std::endl;
```

Comment:
Not sure if it's a good idea to have these prints.

Reply:
I put them for debugging, should have deleted them. Thanks for pointing that out.